src/makeRobot.js

```diff
@@ -38,7 +38,60 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    goForward(value = 1) {
+      if (value > 0) {
+        this.coords.y += value;
+      };
+
+      return this;
+    },
+    goBack(value = 1) {
+      if (value > 0) {
+        this.coords.y -= value;
+      };
+
+      return this;
+    },
+    goRight(value = 1) {
```

Comment:
```suggestion
    },
    
    goRight(value = 1) {
```
Add an empty line between for better readability 👍